Makefile

```diff
@@ -23,13 +17,21 @@ legacy_build_docker:
 all: docker_build_all
 
 #
-# These recipes build all the docker images via a common
-# pattern recipe 'docker_build_%'
+# These recipes build all the top-level docker images
 # TODO grab release tag from env vars
 
-docker_build_%:
+docker_build_%_image:
 	docker build -t ${REGISTRY}/$(IMAGE_NAME) -f $(DOCKERFILE) $(DIR)
 
+#
+# These build the sandbox images and also update (sync) them locally
+# from Docker to podman. This is needed for local analyses; in order
+# to use these updated images, pass 'nopull' to run_analysis.sh
+#
+docker_build_%_sandbox:
+	docker build -t ${REGISTRY}/$(IMAGE_NAME) -f $(DOCKERFILE) $(DIR)
+	sudo buildah pull docker-daemon:${REGISTRY}/${IMAGE_NAME}:latest
```

Comment:
missed this earlier: is sudo strictly necessary here?

Reply:
Yeah it doesn't work without sudo. For reference, the command is from https://github.com/ossf/package-analysis/blob/main/sandboxes/README.md